content/en/docs/concepts/configuration/secret.md

```diff
@@ -13,8 +13,8 @@ weight: 50
 
 {{% capture overview %}}
 
-Kubernetes `secret` objects let you store and manage sensitive information, such
-as passwords, OAuth tokens, and ssh keys.  Putting this information in a `secret`
+Kubernetes Secret objects let you store and manage sensitive information, such
+as passwords, OAuth tokens, and SSH keys.  Putting this information in a Secret
 is safer and more flexible than putting it verbatim in a
 {{< glossary_tooltip term_id="pod" >}} definition or in a {{< glossary_tooltip text="container image" term_id="image" >}}. See [Secrets design document](https://git.k8s.io/community/contributors/design-proposals/auth/secrets.md) for more information.
```

Comment:
@sharjeelaziz, if you wanted to be thorough about tidying the page, you could move:
```markdown
See the [Secrets design document](https://git.k8s.io/community/contributors/design-proposals/auth/secrets.md) for more information.
```
to the _What's next_ section at the end of the page.